tools/agents/doctor.py

```
@@ -0,0 +1,185 @@
+
+
+def runs_scan(wf_dir=WF_DIR):
+    """מתי כל workflow רץ לאחרונה. שלושה מצבים, לא שניים."""
+    out = {}
+    for fn in sorted(os.listdir(wf_dir)):
+        if not fn.endswith((".yml", ".yaml")):
+            continue
+        try:
+            r = subprocess.run(
+                ["gh", "run", "list", "--workflow", fn, "--limit", "1",
+                 "--json", "createdAt,conclusion"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)